daemon/sweep_test.go

```diff
@@ -0,0 +1,136 @@
+package daemon
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/athanorlabs/atomic-swap/coins"
+	"github.com/athanorlabs/atomic-swap/common/types"
+	"github.com/athanorlabs/atomic-swap/monero"
+	"github.com/athanorlabs/atomic-swap/rpcclient"
+	"github.com/athanorlabs/atomic-swap/rpcclient/wsclient"
+	"github.com/athanorlabs/atomic-swap/tests"
+)
+
+// This test starts a swap between Bob and Alice. Bob completes the swap
+// successfully, and then both swapd daemons are shut down before Alice
+// completes the sweep from the swap wallet back to her primary wallet.
+// The Alice restarts, she should not try to sweep again and instead know
+// that the swap is already complete.
+func TestIssue385(t *testing.T) {
```

Comment:
Then Alice restarts ...

Maybe call this: `TestAliceStoppedAndRestartedDuringXMRSweep`